documentation/_Sidebar.asciidoc

```diff
@@ -63,4 +63,5 @@
 
 === Guides
 * link:quarkus/guide-logging.asciidoc[Logging]
-* link:quarkus/guide-cors-support.asciidoc[CORS support]
\ No newline at end of file
+* link:quarkus/guide-cors-support.asciidoc[CORS support]
+* link:quarkus/guide-building-native-image.asciidoc[Native image building]
```

Comment:
actually the includes in the devon4j-doc is more important than the _Sidebar.
If the guide is not included in the main document, then links using `link:` syntax to it will not work in PDF or on website.

Further, just a picky side-note:
Asciidoc files will be linked from multiple places. Therefore the name should be expressive but also brief. So `guide-native-image.asciidoc` would be shorter and more intuitive as your link texts are always "native image":
```
link:quarkus/guide-building-native-image.asciidoc[native-images]
```
Actually we should have also omitted the `guide-` prefix but we do not want to break existing URLs so we have to keep that for consistency.